types/asana/tsconfig.json

```diff
@@ -6,7 +6,7 @@
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
-        "strictNullChecks": false,
+        "strictNullChecks": true,
```

Comment:
This wasn't set per recommendations in the top level README, and as a result the tests weren't as sensitive as they should have been.